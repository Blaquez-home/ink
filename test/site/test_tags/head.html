<meta name='generator' content='Octopress 1.0.0.alpha.24'>
Testing head tag
Testing that content_for is additive

