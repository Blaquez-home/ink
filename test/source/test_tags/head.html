---
layout: head
---
{% head %}
Testing head tag
{% endhead %}

{% head %}
Testing that content_for is additive
{% endhead %}
