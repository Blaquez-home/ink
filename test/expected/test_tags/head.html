<meta name='generator' content='Octopress 1.0.0.alpha.21'>
Testing head tag
Testing that content_for is additive

